.rubocop.yml

```diff
@@ -2,6 +2,16 @@ require:
   - rubocop-rspec
   - rubocop-rails
 
+AllCops:
+  Exclude:
+    - 'spec/rails_helper.rb'
+    - 'spec/spec_helper.rb'
+    - 'db/schema.rb'
+    - 'db/migrate/*'
+    - 'config/**/*'
+    - 'bin/**/*'
+    - '*'
```

Comment:
This change is to exclude any file at root, any path into config/ and bin/, any migration file, and spec_helper and rails_helper